manifests/rootsync-crd.yaml

```diff
@@ -1209,16 +1209,16 @@ spec:
                     description: valuesFileRefs holds references to objects in the
                       cluster that represent values to use instead of default values
                       that accompany the chart. Currently, only ConfigMaps are supported,
-                      and the ConfigMap must be in the same namespace as the RootSync
-                      or RepoSync. When multiple valuesFiles are used, duplicated
-                      keys in later files will override the value from earlier files.
-                      This is equivalent to passing in multiple valuesFiles to Helm
-                      CLI. If `values` is also specified, fields from `values` will
-                      override fields from valuesFileRefs.
+                      and the ConfigMap must be in the same namespace as the RootSync/RepoSync.
+                      When multiple valuesFiles are used, duplicated keys in later
```

Comment:
s/valuesFiles are used/values files are specified/

Reply:
same for valuesFiles below

Reply:
done